### ISSUE TITLE
Large notebooks save fails with failed to fetch

### ISSUE THREAD

**Issue by barperez111** [jupyterhub/zero-to-jupyterhub-k8s#2166]
### Bug description

Hi!
Im running zero to jhub on eks.
when saving large notebooks (~50 mb) the corresponding /api/contents/Huge.ipynb?content=0&1619793 HTTP put request fails with connection reset (after something like 1 minute).
running on a better network the request takes 40sec and file is saved properly 
When saving on The Littlest JupyterHub (on the *bad* network) request succeed after 2 minutes while on zero to jhub i get connection reset after 1 min. 

This led me to think this is some sort of timeout issue, so i have increased timeouts for every component i could think about (the aws load balancer, the configurable http proxy,)

I saw there was an issue regarding getting connection reset on kubernetes as a result of INVALID packets that are not dropped 
(see: https://kubernetes.io/blog/2019/03/29/kube-proxy-subtleties-debugging-an-intermittent-connection-reset/). i tried to configure conntrack to be liberal but that didn't work either. 

any suggestions?

Thnx a lot!

**Comment by consideRatio** [comment]
For this to resolve into a concrete action point, we need to understand the situation even better.

1. Is the 1 minute limitation related to any component of this Helm chart network routing, or does it relate to network routing part of the AWS infrastructure etc?
2. What kind of network traffic is influenced by this etc?

If this issue isn't verified to be related to the Helm chart's components in >1 month, I suggest it is closed and re-opened at the point it is.

**Comment by manics** [comment]
There's a discussion on Discourse: https://discourse.jupyter.org/t/large-notebooks-save-fails-with-failed-to-fetch/9022
I think we should close this issue to prevent discussions getting split, and re-open if it's confirmed to be a bug.

**Comment by barperez111** [comment]
Hi! updating here !

- As suggested [here](https://discourse.jupyter.org/t/large-notebooks-save-fails-with-failed-to-fetch/9022/2)  I added a standalone base notebook app to my kubernetes cluster .
- Using this basic notebook, large notebooks are saved just fine.
- then, added a configurable http proxy in front of the standalone notebook app and the problem reoccurs .

Adding the proxy somehow causes the issue... I tried to increase chp time outs (using flags --timeout and --proxy-timeout) but that didn't help...

Any thoughts?
(updating here cause discourse posts takes long to appear )
Thanks a lot!!!
